src/macros.jl

```diff
@@ -39,21 +39,21 @@ ERROR: LoadError: Error thrown by GAP: Error, no method found! For debugging hin
 [...]
 
 julia> @gap [ 1,, 2 ]
-ERROR: syntax: unexpected ","
+ERROR: syntax: unexpected \",\"
```

Comment:
I had run Documenter.jl manually because of the documentation extension, and it complained about syntax errors. (Docstrings must be strings, this is not the first time when I run into problems because of that; I prefer GAP's way of handling documentation.)
It was surprising for me that this problem had not been detected by the automatic tests.

Reply:
Indeed, that is worrying me.